hw4ParseTree/hw4ParseTree/Addition.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Hw4ParseTree
+{
+    /// <summary>
+    /// класс для сложения
+    /// </summary>
+    public class Addition : Operator
+    {
+        public override char Sign { get; }
```

Comment:
```suggestion
        public override char Sign => '+';
```
И тогда в конструкторе не надо его инициализировать